src/go/rpk/build.sh

```diff
@@ -9,12 +9,23 @@
 # by the Apache License, Version 2.0
 
 version=$1
+root=$(git rev-parse --show-toplevel)
 rev=${2:-$(git rev-parse --short HEAD)}
 
-out_dir="$(go env GOOS)-$(go env GOARCH)"
+go_bin="${root}/vbuild/go/bin/go"
+if [ ! -f "${go_bin}" ]; then
+    # try using the system go
+    if ! command -v go &> /dev/null
```

Comment:
use `if [ ..]; then` in one line.

Reply:
i want to reassign assign go_bin in the case i enter the outer if but not the inner one